js/predictions.md

```diff
@@ -65,7 +65,7 @@ Amplify.addPluggable(new AmazonAIPredictionsProvider());
 
 #### Manual Setup
 
-The manual setup enables you to use your existing Amazon Pinpoint resource in your app.
+The manual setup enables you to use your existing Amazon Cognito Identity Pool resource in your app.
```

Comment:
```suggestion
The manual setup enables you to use your existing Amazon AI and ML resources in your app.
```

Reply:
Thanks for pointing this out, the idea is that if you have already configured permissions (IAM roles and policies on your identity pool to have access to AI services) you can manually configure those on the library

Reply:
That makes sense. I was using Cognito as that was the only configurable parameter in the code section below, but if you are using Rekognition, you may want to point to your own face library and Cognito would be used from a different category. So what you wrote is better.

Reply:
@jodion can you commit the suggestion so I can merge the pr. Thanks!